micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/correlationid/CorrelationIdFilter.groovy

```diff
@@ -61,16 +80,21 @@ class CorrelationIdFilter extends OncePerRequestFilter {
         return correlationId
     }
 
-    //TODO: add microservice id to corrId, so that we know where it was created
-    private String createNewCorrIdIfEmpty(String currentCorrId) {
-        if (!hasText(currentCorrId)) {
-            currentCorrId = UUID.randomUUID().toString()
-            MDC.put(CORRELATION_ID_HEADER, currentCorrId)
-            log.debug("Generating new correlationId: " + currentCorrId)
-        }
+    private String createNewCorrIdIfEmpty() {
```

Comment:
Maybe [`CorrelationIdUpdater.updateCorrelationId()`](https://github.com/4finance/micro-infra-spring/blob/master/micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/correlationid/CorrelationIdUpdater.groovy#L23) could be used?